### DISCUSSION TITLE
Tasks Kanban view assigned user list

### DISCUSSION THREAD

**Discussion by elmoandzoey** [Leantime/leantime#858] [General]
First, let me say Leantime is an awesome product that we didn't know we were missingn until we found it! We're using the Kanban view for the tasks as overview. At the bottom right of each is task is a around icon which shows the icon of the assigned team member. However, if you click on it, it shows a list of all team members that have access to the project as being the assigned member for the task and does not highlight the sole team member who's really assigned. It has caused a bit of a problem for us as on first view you can't make out who the user is if the users have not assigned themselves an icon or if the icons are similar. Neither is it marked in the popup lost which team member is assigned this one task, it just shows which you can assign. You have to open each task and look in the sidebar to figure out for sure who the assigned team member is. Is there a way to fix this and just show the one team member assigned to the task? 

Pictures for illustration:
[1] 3 different assigned team members, can't really make out who it is just by looking at the icon [ETA: Maybe its possible to show the initials with the icon to make it clearer who's assigned instead of just the icon?)
![grafik](https://user-images.githubusercontent.com/41942236/184099856-95b16548-b156-4398-8a0b-c12bb2f17e75.png)

[2] User list when clicking on the icon shows all team members working for the project but does not highlight the assigned team members for the task
![grafik](https://user-images.githubusercontent.com/41942236/184100138-b12893cb-7507-4052-8343-15cad8ba6f4d.png)

Listview shows the name of the team member no problem but the popup that appears when clicking on the name has the same problem in that it shows all team members working on the project as the assigned team member for the task.

[1] Listview showing the firstname of the assigned team member for the task
![grafik](https://user-images.githubusercontent.com/41942236/184100551-ac88ba98-3b8d-4a0f-8575-0eb777fc9e06.png)

[2] Popupmenu with the same problem; showing all project team members but does not highlight the one currently assigned for the task
![grafik](https://user-images.githubusercontent.com/41942236/184100841-dc7a67f4-7a30-4046-9adc-2ceecd244d53.png)

**Comment by marcelfolaron** [comment]
Hey,

Sorry for the delay. So the main confusion might come from the translation. It should be 
“Zuweisen an”

the users that pop up below are users from the project, not assigned users.
The assigned user is the one picture shown on each card. We should highlight the user though in the Dropdown- as well

**Comment by elmoandzoey** [comment]
Thank you! I was wondering if it was a translation issue in that it was supposed to be "Zuweisen an" instead of "Zugewiesen an". We have since made icons with people's initials as their personal icons since that makes it easier to see who is assigned than trying to figure out which person's colorful icon you are currently seeing. Some of our team members have problems to identify the small icon properly if it's a picture. But if the person who is currently assigned is highlighted or marked in the dropdown, it would be very helpful.

**Comment by marcelfolaron** [comment]
thanks, I agree.

i am taking two action items:
- set the default profile image to color + initial
- highlight who is assigned in the dropdown
- increase size of icon on task cards
